zanata-war/src/main/webapp/WEB-INF/layout/language/request_join_modal.xhtml

```diff
@@ -0,0 +1,114 @@
+<div xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:h="http://java.sun.com/jsf/html"
+  xmlns:s="http://jboss.org/schema/seam/taglib"
+  xmlns:rich="http://richfaces.org/rich"
+  xmlns:a4j="http://richfaces.org/a4j"
+  xmlns:zanata="http://java.sun.com/jsf/composite/zanata"
+  class="modal" id="joinLanguageDialog" tabindex="-1" role="dialog"
+  aria-labelledby="myModalLabel" aria-hidden="true">
+
+  <script>
+    $(document).ready(function() {
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-emailType')}')).
+        val('#{languageJoinAction.emailType}');
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-subject')}')).
+        val('#{languageJoinAction.subject}');
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-id')}')).val('#{id}');
```

Comment:
Why are all these set with jQuery instead of just putting them in the template directly?